src/MyClass.java

```diff
@@ -0,0 +1,8 @@
+public class MyClass {
```

Comment:
why did you delete the method that finds the sum of two numbers? Revert it pls

    static int sumOfTwoNumbers(int a, int b){
        int sum = a + b;
        return sum;
    }